### ISSUE TITLE
Some skins lose decal textures

### ISSUE THREAD

**Issue by alazi** [acidbubbles/vam-improved-pov#6]
When using a skin (eg male 2) that uses the GlossNMDetailCullComputeBuff shader you lose the decal texture in reflections, since the transparent version doesn't have that. I'm not sure there is a great solution here but swapping the shader and broadcasting OnAppFocus each frame works, though I don't know if it might have a small performance impact. (I'm guessing the broadcast is required and sufficent because changing shader breaks the bindings, but the reflection cameras obviously have to run before the main camera, and we don't actually need to render anything on the main camera)

**Comment by acidbubbles** [comment]
Mmmh that's a very good point, the plugin is trying very hard not to affect performance and broadcasting an operation like this would definitely do. A much better solution would be for this feature to be built-in VaM :(

**Comment by VamDazzler** [comment]
@alazi Have you tried the "Render Queue" slider? It's on the "shader" tab of the material and I've often found that some materials benefit from being shuffled around. It may be that the shader you're using is being run /after/ the mirror reflection cameras and so wouldn't be picked up by them.

**Comment by alazi** [comment]
No, the skin shows up, it's just that the shader used by skins varies, and the equivalent transparent ones vary as well, and this particular one literally does not have a decal texture parameter unlike the others. There's other workarounds of course - "just use a different skin" and "clothing" that's just the g2 mesh.
